apps/files/src/actions/convertUtils.ts

```diff
@@ -77,8 +77,8 @@ export const convertFiles = async function(fileIds: number[], targetMimeType: st
 
 			// We already check above when all files failed
 			// if we're here, we have a mix of failed and successful files
-			showError(t('files', '{count} files could not be converted', { count: failed.length }))
-			showSuccess(t('files', '{count} files successfully converted', { count: fileIds.length - failed.length }))
+			showError(t('files', failed.length === 1 ? '{count} file could not be converted' : '{count} files could not be converted', { count: failed.length }))
+			showSuccess(t('files', fileIds.length === 1 ? '{count} file successfully converted' : '{count} files successfully converted', { count: fileIds.length - failed.length }))
```

Comment:
This does not work as you cannot have conditions in translations - this will not be extracted correctly.
Also it should use the plural form `n`:

```suggestion
			showError(n('files', 'one file could not be converted', '%n files could not be converted', failed.length))
			showSuccess(n('files', 'one file successfully converted', '%n files successfully converted', fileIds.length - failed.length))
```

Reply:
None of the above, `failed.length === 1` is already handled 10 lines above

Reply:
yes but not the difference between how plural is handled in different languages.
Not every language has the same form for plural.